handbook/company/customer-first.md

```diff
@@ -33,7 +33,7 @@ This is how this all fits together in terms of workflow:
 	* Help troubleshooting something not working correctly
 
 
-Currently, CEs are responsible for answering all how-to questions and handling all feature requests (whether these come up pre- or post-sales). Similarly, CS is responsible for troubleshooting something not working correctly (whether these come up pre- or post-sales), taking into account the prioritization considerations as provided by CEs, and (effective 2021-08-08) adhering to our contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas) post-sales and 1 hour internal SLA response time pre-sales between 2am-5pm PT Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22) -- between now and 2021-08-08, as the CS team hires and onboards additional folks, they will adhere to a 24 hour SLA for all pre-sales issues between 9am-5pm PT Monday-Friday. As CS continues to hire during FYQ2 and new folks are onboarded, CS will also take on responsibility for how-to questions and feature requests for non-strategic customers.
+Currently, CEs are responsible for answering all how-to questions and handling all feature requests (whether these come up pre- or post-sales). Similarly, CS is responsible for troubleshooting something not working correctly (whether these come up pre- or post-sales), taking into account the prioritization considerations as provided by CEs, and adhering to our contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas) post-sales and 1 hour internal SLA response time pre-sales between 2am-5pm PT Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22). As CS continues to hire during FYQ2 and new folks are onboarded, CS will also take on responsibility for how-to questions and feature requests for non-strategic customers.
```

Comment:
> As CS continues to hire during FYQ2 

Is FYQ2 correct or FY22?

Reply:
It is actually FYQ2 ... Aimee and are still working out the details, but the hiring we did is in support of that.

Reply:
just making sure :D approved! Thanks V!